src/components/ConfigEditor/ConfigHelp.tsx

```diff
@@ -1,10 +1,24 @@
-import React from 'react';
+import { DataSourceSettings } from '@grafana/data';
+import { ConfigSection } from '@grafana/experimental';
+import React, { useMemo } from 'react';
+import { AdxDataSourceOptions, AdxDataSourceSecureOptions } from 'types';
+import { isCredentialsComplete } from './AzureCredentials';
 
-interface ConfigHelpProps {}
+interface ConfigHelpProps {
+  options: DataSourceSettings<AdxDataSourceOptions, AdxDataSourceSecureOptions>;
+}
+
+const ConfigHelp: React.FC<ConfigHelpProps> = ({ options }) => {
```

Comment:
I believe we're still trying to [avoid](https://github.com/grafana/grafana/blob/main/contribute/style-guides/frontend.md#how-to-declare-functional-components) using `React.FC`. So I think this should be something like this:

```suggestion
function ConfigHelp({ options }: ConfigHelpProps) {
```

Reply:
To avoid too many changes would you be okay if I left this for a separate refactor? I think we're actually doing this for most of the functional components so I'd like to do them all at once in a clean PR, how does that sound?

Reply:
:+1: sounds good to me